src/commands/Respond.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const Command = require('../Command.js');
+const md = require('node-md-config');
+
+/**
+ * Describes the Armor command
+ */
+class Respond extends Command {
+  /**
+   * Constructs a callable command
+   * @param  {Logger}           logger                The logger object
+   * @param  {string}           [options.prefix]      Prefix for calling the bot
+   * @param  {string}           [options.regexPrefix] Escaped prefix for regex for the command
+   * @param  {MarkdownSettings} [options.mdConfig]    The markdown settings
+   * @param  {Database}         [options.settings]    Settings for this bot
+   */
+  // eslint-disable-next-line no-useless-escape
+  constructor(logger, { mdConfig = md, regexPrefix = '\/', prefix = '/', commandHandler = null } = {}) {
+    super(logger, { mdConfig, regexPrefix, prefix, commandHandler });
+    this.commandId = 'settings.response';
+    // eslint-disable-next-line no-useless-escape
+    this.commandRegex = new RegExp(`^${regexPrefix}toggle response$`, 'i');
+    this.commandHelp = `${prefix}toggle response | Toggle response to tsettings changes in this channel `;
+    this.requiresAuth = true;
+    this.blacklistable = false;
+  }
+
+  run(message, { settings = null, stringManager = null } = {}) {
+    if (settings !== null) {
+      settings.getChannelRespondToSettings(message.channel.id)
```

Comment:
Oh, I see. I still think it could be rearranged a bit in order to minimize the number of nested `then`s, for example by making the success callbacks return promises.
Don't the two `settings.getChannelRespondToSettings(message.channel.id)` lines do the same thing?